src/client/powerpages/Contentsnippet.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
ContentSnippet does not looks descriptive. How we name it as ContentSnippetOperations